src/containers/Cluster/ClusterInfo/ClusterInfo.tsx

```diff
@@ -158,36 +186,30 @@ export const ClusterInfo = ({
         return <ResponseError error={error || nodesError} />;
     }
 
+    let internalLink = backend + '/internal';
+
+    if (singleClusterMode && !customBackend) {
+        internalLink = `/internal`;
+    }
+
+    const {info = [], links = []} = additionalClusterProps;
```

Comment:
Inside `getInfo` there are also fallbacks. It seems to be enough callbacks inside the function.

Reply:
I use spread operator on `links` below. So it's needed here as well. Removed fallbacks from `getInfo`